packages/react-core/src/components/TextInput/TextInput.tsx

```diff
@@ -26,15 +26,22 @@ export enum TextInputReadOnlyVariant {
   plain = 'plain'
 }
 
+interface TextinputExpandedObj {
+  isExpanded: boolean;
+  ariaControls: string;
+}
```

Comment:
Can we do the following:

- (nit) update the name to `TextInputExpandedObj` (input is lowercased currently)
- export the interface
- add it to the props table
- add descriptions for the props

Not sure if it's common in other components, but also thinking the `ariaControls` should be kebab cased like other basic aria props (`aria-labelledby` and such), but not a blocker